content/copilot/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Docker for GitHub Copilot
+description: Learn how to use the Docker Extension for GitHub Copilot to streamline Docker-related tasks directly within GitHub and Visual Studio Code.
+keywords: Docker, GitHub Copilot, extension, Visual Studio Code, chat, ai, containerization
+sitemap: false
+---
+
+> **Early Access**
+>
+> The Docker for GitHub Copilot extension is in [Early Access](../../release-lifecycle.md).
```

Comment:
This is tricky because it's part of GitHub's limited public beta. Not sure if we want to mention that. We are beholden to their timelines on public preview etc.

Reply:
Yeah let's mention that! I will update.